app/javascript/article-form/elements/moreConfig.jsx

```diff
@@ -1,8 +1,17 @@
 import { h, Component } from 'preact';
 import PropTypes from 'prop-types';
-import { generateMainImage } from '../actions';
 
-// const ImageManagement = ({ onExit }) => (
+const TextField = ({ label, id, value, onKeyUp }) => {
```

Comment:
@nickytonline shouldn't this component have props as well?

Reply:
I'm assuming you mean PropTypes? If so, potentially. I don't think I've ever seen prop types on a component that isn't exported. For context see this issue, https://github.com/yannickcr/eslint-plugin-react/issues/494#issuecomment-230982532

Reply:
Updated Proptypes as component as well @rhymes  @nickytonline